cmd/git-sync/main.go

```diff
@@ -62,6 +63,9 @@ var flMaxSyncFailures = flag.Int("max-sync-failures", envInt("GIT_SYNC_MAX_SYNC_
 var flChmod = flag.Int("change-permissions", envInt("GIT_SYNC_PERMISSIONS", 0),
 	"the file permissions to apply to the checked-out files")
 
+var flWebhooks = flag.String("webhook", envString("GIT_SYNC_WEBHOOK", ""),
```

Comment:
As per comments in #110 I think it's net simpler to support just one webhook and to do so via 3 flags:

string URL (default "", meaning no webhook)
string method (default "POST")
int success (default 200)

Reply:
Moved to all separate CLI flags